test/mock_data/get_users_2.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Why is this file called `get_users_2.json` ? I believe that just  `user.json` is probably better, I cannot find the meaning of the number 2, and the `get` I believe is not necessary.